### PR TITLE
path from query

### PR DESCRIPTION
**Pull request by ozanturksever**
ability to pass paths from query

### CODE REVIEW

**Comment by de-robat** [comment]
ill merge it and add some tests to clarify the usage.

File: src/index.js [hunk 1]
```diff
@@ -13,13 +13,16 @@ module.exports = function (source) {
...
+	var paths = query.path || [];
 
 	var callback = this.async();
 	this.cacheable && this.cacheable();
 
 	var str = source,
 		lessVars = {},
-		config = {};
+		config = {
+			paths: path
```
Comment (de-robat) [inline src/index.js:24]: typo, discoverd while testing

**Comment by de-robat** [comment]
done.

**Comment by ozanturksever** [comment]
great thanks.

looking for a npm update.

**Comment by de-robat** [comment]
i already did it the very same day, so you should be good to go :)

**Pull request merged**